nzic/index.md

```diff
@@ -4,30 +4,30 @@ layout: nzic_page
 nzic_weight: 0
 ---
 
-The NZIC contest series, run by the New Zealand Olympiad in Informatics, will be held again this year. There will be 4 regular rounds held throughout the year. The difficulty ranges from entry-level problems to more challenging problems. To practice, sign up at [https://train.nzoi.org.nz/](https://train.nzoi.org.nz/).
+The NZIC contest series, run by the New Zealand Olympiad in Informatics, will be held again in 2017. There will be 3 regular rounds held throughout the year. The difficulty ranges from entry-level problems to more challenging problems. To practice, sign up at [https://train.nzoi.org.nz/](https://train.nzoi.org.nz/).
```

Comment:
Could it be helpful to explicitly say that the contest is run on train.nzoi and problems will follow a similar format?